torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4547,6 +4547,22 @@ def _tensor(shape, dtype=dtype, low=None, high=None):
 
     return [SampleInput(tensor, args=args) for tensor, args in test_cases]
 
+def sample_inputs_mse_loss(op_info, device, dtype, requires_grad, **kwargs):
+    def make_tensor_(shape):
+        return make_tensor(shape, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    sample_inputs = [
```

Comment:
This might be simpler and more readable if it enumerated a product of shapes and reduction types instead of creating a list and then extending it